.github/dependabot.yml

```
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: pip
+    directories:
+      - /utils/cq/requirements.txt
+      - /requirements-build.txt
+      - /requirements-ftest.txt
+      - /requirements-utest.txt
```

I'm not sure about this. The key is "directories" but I'm trying to include specific "files".
I think I might instead need
```suggestion
    directories:
      - /utils/cq/
      - /
```

---

Oh, hrm.  Will Dependabot know about `requirements-*.txt` files?

---

Apparently not :/ The file has to end with "requirements.txt"

---

But I wonder if it will look at `/requirements.txt` and recursively drill down?